lib/content.js

```diff
@@ -33,6 +34,11 @@ function isBodyEqual( httpReq, specReq, contentType ) {
         return true;
     }
 
+    if (contentType.indexOf('application/x-www-form-urlencoded') > -1) {
```

Comment:
I prefer to use the RegExp like comparison to avoid any case on the header strings.
`/^application\/x-www-form-urlencoded/i.test(contentType)`